docs/use-cases/organizational-access-control.mdx

```diff
@@ -0,0 +1,38 @@
+---
+
+<!--todo: description --->
+
+---
+
+# Organizational Access Control
+
+## Abstract
+
+Traditional organizations mainly have a distribution of access to resources based on the roles.
+DAOs often do not have such a mechanism, but both types of organizations suffer from similar problems, mainly related to privacy.
+The most obvious ones are:
+- Members may fear retaliation for reporting misconduct or unethical behavior within the organization.
+- Online forums, clubs, or social groups may face challenges with members expressing their opinions openly due to fear of judgment or repercussions.
+- Members may be hesitant to provide candid responses if they fear their personal information will be disclosed.
+- Activists involved in political movements may face risks to their safety or privacy when expressing dissenting opinions.
+- DAOs suffer from the lack of private proposals, voting, and resource distribution.
+
+Even without these problems, members would like to access files, vote, propose without revealing their private data, such as e-mail address, first and last name, telephone number, etc.
+
+![Rarimo social graphs scheme](/img/social-graphs-links-scheme.png)
+
+Many organizations are public in nature, e.g. volunteer organizations.
```

Comment:
"volunteer organizations" is a literal translation, use something like "non-profit oranizations"